var/spack/repos/builtin/packages/fftw/package.py

```diff
@@ -157,6 +105,17 @@ def autoreconf(self, spec, prefix):
             autoreconf = which('autoreconf')
             autoreconf('-ifv')
 
+    @property
+    def selected_precisions(self):
+        """Precisions that have been selected in this build"""
+        spec = self.spec
+        return list(filter(None, [
+            'float' if spec.satisfies('precision=float') else None,
+            'double' if spec.satisfies('precision=double') else None,
+            'long_double' if spec.satisfies('precision=long_double') else None,
+            'quad' if spec.satisfies('precision=quad') else None
+        ]))
```

Comment:
I'm not super-familiar with multi-valued variants, but couldn't this just be written as
```python
return list(self.spec.variants['precision'].value)
```
?

Reply:
Good catch, we don't even need to wrap it in `list`.